design/idempotency_ids.md

```diff
@@ -49,10 +59,9 @@ in that id and the cluster can reclaim the space used to store the idempotency
 id. The commit proxy that committed a batch is responsible for cleaning all
 idempotency kv pairs from that batch, so clients must tell that specific proxy
 that they're done with the id. The first proxy will also periodically clean up
-the oldest idempotency ids, based on a policy determined by two knobs.  One knob
+the oldest idempotency ids, based on a policy determined by knobs.  One knob
 will control the minimum lifetime of an idempotency id (i.e. don't delete
```

Comment:
Should we add the knob here?

Also, nit: i.e. --> for example

Reply:
> Should we add the knob here?

Done

> Also, nit: i.e. --> for example

"i.e." means "that is" and "e.g." means "for example". I really do mean "i.e." here, so I'd be willing to change to "that is" if you think "i.e." is too arcane.

Reply:
This is totally fine. I'd suggested "example" because the knob doesn't have to be set for 1 day, but this is a minor/subjective stylistic choice, and "i.e." is absolutely clear too!